lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -39,6 +39,15 @@ using llvm::cast;
 using ArgumentDictionaryTy =
     std::unordered_map<std::string, const caffe2::Argument *>;
 
+/// For the quantized Caffe2 ops, the activations are quantized to uint_8.
+/// In Glow, the activations are quantized to int_8. Therefore, for the offset
+/// read from quantized caffe2 model, we need to subtract 128(i.e. INT8_MIN) to
+/// make the activations becomes int8_t.
+/// For Glow: -127 <= orig_fp32/scale_1 + offset_1 < 128
+/// For Caffe2: 0 <= orig_fp32/scale_2 + offset_2 < 255
+/// Therefore, we can make scale_1 == scale_2, and offset_1 = offset2 - 128
+#define OFFSETSHIFT 128
```

Comment:
can you make a constant instead of `define`

Reply:
@rdzhabarov In the quantized mode, for Int8MaxPool, the scale of input and output are the same. In Caffe2 implantation,  for INt8MaxPool, even if there is a pre-chosen quantization parameters for the output, it is ignored because maxpool output quantization should be same as the input. So I don't think we need to fix our backend implementation :)

Reply:
Hm, i'm a bit confused. You modified the verifiers for pools
> isValid &= checkTypeIgnoreShape(src, dest, dest.getNode());

This used to check that scale and offset is the same, not it does not.
Missing anything?

Reply:
@rdzhabarov Actually, this one was triggered by Int8AvgPool, so  I modified it.

Reply:
Both maxpool and avgpool would likely change input's floating point dynamic range.
I do not think it's correct to assume that inputs/outputs have the same scale.
However, these changes are not directly tight to this PR as it was the case we had this problem before.

Can you file issue to fix int8 int/max pool and fix related code (quantization procedure, loading of c2, etc).

Reply:
For quantized avgpool, our current implantation does't require the input and output has the same scale and offset. So you mean for Int8MaxPool, we need to handle the input and output with different scale?

Reply:
Yes, both operations need to handle diff {scale,offset} for input/output.

Reply:
After discussed offline with @rdzhabarov, for maxpool, we will keep the quantized input and output have the same params.